### DISCUSSION TITLE
nvim-treesitter is not installing the parsers

### DISCUSSION THREAD

**Discussion by LaBatata101** [nvim-treesitter/nvim-treesitter#2406] [Q&A]
Since I switched from using [vim-plug](https://github.com/junegunn/vim-plug) to [packer.nvim](https://github.com/wbthomason/packer.nvim) I noticed that nvim-treesitter is not installing the parsers even though it prompts that is downloading and installing.

This is how I'm installing nvim-treesitter with packer.nvim:
```lua
  use {
    'nvim-treesitter/nvim-treesitter',
    run = ':TSUpdate'
  }
```
Here's my nvim-treesitter configuration:
```lua
require'nvim-treesitter.configs'.setup {
    highlight = {
        enable = true
    },
    ensure_installed = {
        "python",
        "rust",
        "vim",
        "lua"
    }
}
```
Every time I open neovim it prompts that is downloading and installing the languages parsers listed in `ensure_installed`. When I run the command `:TSInstallInfo` none of those languages show up as installed.

If I install nvim-treesitter with vim-plug everything works normal.
I don't know if this is a bug or something wrong in my configuration.

**Comment by theHamsta** [comment]
Do you use the PackerCompile feature? It is working fine for mit with packer

**Reply by LaBatata101** [reply]
Yes, I use PackerCompile.
I don't what happened but after a few reinstalls nvim-treesitter started working again.

**Reply by theHamsta** [reply]
It's possible that your config wasn't executed and a manual :PackerCompile solved it. At least, I stumbled a few times over that, that PackerCompile freezes the config to the point it was executed.
